src/main/java/com/capitalone/dashboard/collector/SCMHttpRestClient.java

```diff
@@ -21,27 +21,31 @@
 public class SCMHttpRestClient {
   private static final Log LOG = LogFactory.getLog(SCMHttpRestClient.class);
 
-  @Inject
-  private GitSettings settings;
+    /*@Inject
+    private GitSettings settings;*/
 
   @Inject
   private RestTemplate restTemplate;
 
-  public ResponseEntity<String> makeRestCall(URI uri, String userId, String password) {
-    String id = userId;
-    String secret = password;
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("GET " + uri);
-    }
-    // fallback to global credentials
-    if (StringUtils.isEmpty(userId) || StringUtils.isEmpty(password)) {
-      id = settings.getUsername();
-      secret = settings.getPassword();
-    }
-    // Basic Auth only.
-    if (!"".equals(id) && !"".equals(secret)) {
-      return restTemplate.exchange(
-          uri, HttpMethod.GET, new HttpEntity<>(createHeaders(id, secret)), String.class);
+    public ResponseEntity<String> makeRestCall(URI uri, String userId, String password) {
+        String id = userId;
+        String secret = password;
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("GET " + uri);
+        }
+        // fallback to global credentials
+        /*if (StringUtils.isEmpty(userId) || StringUtils.isEmpty(password)) {
+            id = settings.getUsername();
+            try {
+                secret = new String(Base64.decodeBase64(settings.getPassword()));
+            } catch (Exception e) {
+                LOG.error(e.getMessage());
+            }
+        }*/
+        // Basic Auth only.
+        if (!"".equals(id) && !"".equals(secret)) {
```

Comment:
Can we use `StringUtils.isNotEmpty()` here?

Reply:
done